Defined maintainers and linked it with the role of maintainers. (#1400)
conda-forge/conda-forge.github.io: src/maintainer/adding_pkgs.rst

Contributing packages
*********************

> A maintainer is an individual who is responsible for maintaining and updating one or more feedstock repositories and packages as well as their future versions. They have push access to the feedstock repositories of the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it. See `Maintainers Role. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__

Reviewer: just to be more specific it should be good to point out that they have push rights only for the packages they maintain.

Author: That is, I should write : They have push access to the feedstock repositories of only the packages they maintain?

Reviewer: I think we can follow the Governance guidelines here:

Author: They have push access to the feedstock repositories of only the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it.

I think this sentence is alright and complies with what you suggested. What do you say?

> A maintainer is an individual who is responsible for maintaining and updating one or more feedstock repositories and packages as well as their future versions. They have the push access to the feedstock repositories of the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it. See `Maintainers Role. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__

Reviewer 2: ```suggestion
A maintainer is an individual who is responsible for maintaining and updating one or more feedstock repositories and packages as well as their future versions. They have push access to the feedstock repositories of the packages they maintain and can merge `PR <https://conda-forge.org/docs/misc/00_intro.html#glossary>`__ into it. See `Maintainers Role. <https://conda-forge.org/docs/maintainer/adding_pkgs.html#maintainer-role>`__
```

> Once you create a package, you automatically become the maintainer of it. In case you wish to be a maintainer of a certain package, you can ask for permission from the current maintainers of it and get yourself added to the list of maintainers. 

Reviewer: ```suggestion
Once you create a package, as a code owner you automatically become the maintainer of it. In case you wish to be a maintainer of a certain package, you can ask for permission from the current maintainers of it and get your `gitub-id` added to the `recipe-maintainers` section in the recipe’s `meta.yaml`. Please refer to `Updating the maintainer list <https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list>'__ for detailed instructions.
```

> Once you create a package, you automatically become the maintainer of it. In case you wish to be a maintainer of a certain package, you can ask for the permission from the current maintainers of it and get youself added in the list of maintainers. 

Reviewer 2: ```suggestion
Once you create a package, you automatically become the maintainer of it. In case you wish to be a maintainer of a certain package, you can ask for permission from the current maintainers of it and get yourself added to the list of maintainers. 
```

Author: Thanks :)

> The list of maintainers of the feedstock package is recorded in the recipe of that package itself.

Reviewer: ```suggestion

```

> Once you create a package, as a code owner you automatically become a maintainer of it. In case you wish to be a maintainer of a certain package, you can ask for permission from the current maintainers of it and get your `gitub-id` added to the `recipe-maintainers` section in the recipe’s `meta.yaml`. `Please refer to Updating the maintainer list <https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list>`__ for detailed instructions.

Reviewer: ```suggestion
The list of maintainers of a feedstock package is recorded in the recipe of that package itself. Once you create a package, as a code owner, you automatically become a maintainer of it. In case you wish to be a maintainer of a certain package, you should ask for permission from the current maintainers of it and get your `gitub-id` added to the `recipe-maintainers` section in the recipe’s `meta.yaml`. 
`Please refer to Updating the maintainer list <https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list>`__ for detailed instructions.
```

> The list of maintainers of the feedstock package is recorded in the recipe of that package itself.

Reviewer: ```suggestion
```
